### DISCUSSION TITLE
private and publics repos

### DISCUSSION THREAD

**Discussion by Irissuu** [community/community#127757] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

Is there any way to present both public and private repository data in the profile? or just 1?

**Comment by davevad93** [comment, marked as answer]
Hi @Irissuu ,

If by present both `public` and `private` repos data in the profile you mean displaying its contents or stats then the answer is no, you can do it just with `public` ones. Private repos are only accessible to you or to people with whom you explicitly share access, but any data will be private and will not be displayed in the profile. For instance, projects that allow you to showcase your stats just as [github-profile-trophy](https://github.com/ryo-ma/github-profile-trophy) or [github-readme-stats](https://github.com/anuraghazra/github-readme-stats) only support showing data from `public` repos.

EDIT: I forgot to mention that the contributions for `private` repos can be displayed in the contribution graph of the profile if you activate the `Include private contributions on my profile` option in the `Settings`. That's the only data from `private` repos that can be displayed on the profile.

![0](https://github.com/community/community/assets/94308234/580318c4-c943-403d-a907-04b70987d2f9)

**Reply by davevad93** [reply]
Correction: [github-profile-trophy](https://github.com/ryo-ma/github-profile-trophy) allows to display some `private` repos stats such as `commits`, `pull requests` and `stars` but [github-readme-stats](https://github.com/anuraghazra/github-readme-stats) none.

**Comment by Irissuu** [comment]
thank you very much! I understand perfectly

**Reply by davevad93** [reply]
You're welcome @Irissuu , happy to help!
